python/lsst/ts/xml/data/sal_interfaces/ESS/ESS_Events.xml

```diff
@@ -1,61 +1,69 @@
+    <Description>Lightning strike, if detected.</Description>
     <item>
       <EFDB_Name>sensorName</EFDB_Name>
-      <Description>The name of the sensor.</Description>
+      <Description>The user-defined name of the sensor (e.g. AuxTel LD-250).</Description>
       <IDL_Type>string</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>strike</EFDB_Name>
-      <Description>Lighting strike alarms, as a mask with numStrikeBits bits. 1 means an alarm is active. Higher-order bits are for strikes that are closer, and thus more worrisome. Each alarm will eventually reset itself if no further activity in that alarm range is detected.</Description>
-      <IDL_Type>unsigned short</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
+      <EFDB_Name>correctedDistance</EFDB_Name>
+      <Description>Distance of the strike, corrected by the hardware.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>km</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>highField</EFDB_Name>
-      <Description>High electrical field alarms, as a mask with numHighFieldBits bits. 1 means an alarm is active. Higher-order bits are for larger electrical fields, and thus are more worrisome. Each alarm will eventually reset itself if no further activity in that alarm range is detected</Description>
-      <IDL_Type>unsigned short</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
+      <EFDB_Name>uncorrectedDistance</EFDB_Name>
+      <Description>Distance of the strike, not corrected by the hardware.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>km</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>numStrikeBits</EFDB_Name>
-      <Description>The maximum number of strike bits.</Description>
-      <IDL_Type>unsigned short</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
+      <EFDB_Name>bearing</EFDB_Name>
+      <Description>Bearing of the strike.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>degree</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>numHighFieldBits</EFDB_Name>
-      <Description>The maximum number of high field bits.</Description>
-      <IDL_Type>unsigned short</IDL_Type>
-      <IDL_Size>1</IDL_Size>
+      <EFDB_Name>level</EFDB_Name>
+      <Description>Alarm level, an AlarmLevel enum limited to Red, Orange and Yellow.</Description>
+      <IDL_Type>int</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>ESS</Subsystem>
+    <EFDB_Topic>ESS_logevent_highElectricField</EFDB_Topic>
+    <Description>High electric field, if exceeding an intensity threshold for a certain amount of time. The thresholds for Red, Orange and Yellow and the time limit can be configured in ts_config_ocs.</Description>
```

Comment:
Same comment as strike: do you plan to time this out after a time? Can there be more than one "high field" event active at the same time (in which case you may need to add some kind of ID field).

Reply:
See my previous comment.